### PR TITLE
Fixes #1319. Made popup open after close window command

### LINKED ISSUE #1319: "Close window" closes Firefox window without confirmation
### Prerequisites:
Mic permissions are enabled.
"Allow" or "Don't Allow" button from “Allow Firefox Voice to Collect Voice Transcriptions” pop-up needs to be clicked.
Several tabs are opened.

### STR:
1. Use the shortcut or click on the mic icon from the browser toolbar.
2. Say or write the "Close window" or "Close this window" command.

### Expected result:
Firefox window tries to close and a confirmation to close the tabs is displayed.

### Actual result:
Firefox window closes without the confirmation.

### Notes:
Reproduced on Mac 10.14 and Win10x64 with Firefox Nightly 76.0a1 (2020-03-17).

### PR DESCRIPTION
**Pull request by saucekode**
Before submitting a final PR, please:

- [ ] Run `npm test` on your machine
- [ ] Run `npm run format` to keep code formatted consistently
- [ ] Reference the issue you are fixing in at least one of your commit messages, as `Fixes #X`

### CODE REVIEW

File: extension/popup/popupView.jsx [hunk 1]
```diff
@@ -633,11 +631,11 @@ const SearchResultsContent = ({
   return (
     <React.Fragment>
       <TextDisplay displayText={displayText} />
-      <div id="search-results">{renderCard()}</div>
+      <div id="search-results">{ transcript.toLowerCase() == 'Hello' ? HelloCard() : renderCard()}</div>
       <div id="search-footer">
         <IntentFeedback
           onSubmitFeedback={onSubmitFeedback}
-          eduText={card && card.answer ? card.answer.eduText : null}
+          eduText={card && card.answer ? card.answer.edutext : null}
```
Comment (ianb) [inline extension/popup/popupView.jsx]: Did you intend to change the case of `eduText`?

File: extension/intents/self/self.js [hunk 2]
```diff
@@ -11,7 +11,22 @@ intentRunner.registerIntent({
 intentRunner.registerIntent({
   name: "self.openLexicon",
   async run(context) {
-    await browserUtil.openOrActivateTab("/views/lexicon.html");
+    context.keepPopup()
+    const helloImage = '../../assets/images/hello-card.jpg'
+    const helloCard = {
+      answer: {
+        imgSrc: `${helloImage}`,
+        helloText: "Is it me you're looking for?",
```
Comment (ianb) [inline extension/intents/self/self.js]: Could you just use `eduText` or `context.displayText()` for this message?

File: extension/popup/popupView.jsx (hunk 1 above)
Comment (saucekode) [reply extension/popup/popupView.jsx]: Hello @ianb was testing out something? That was a mistake, already fixed it on my local machine. Sorry about this.

File: extension/intents/self/self.js (hunk 2 above)
Comment (saucekode) [reply extension/intents/self/self.js]: Okay, will do. Just thought helloText seemed appropriate for the context. Thanks. Will make the changes ASAP

**Comment by saucekode** [comment]
Did a better fix for close window intent, @alexandra-martin made a complaint. Do check it out @ianb

**Pull request closed without merging**